text/serialization.tex

```diff
@@ -156,6 +156,7 @@ \subsection{Block Serialization}
   ) \\
   \se(x \in \mathbb{C}) &\equiv \se(x_\mathbf{y}, x_r) \\
   \se(x \in \defxfer) &\equiv \se(\se_4(x_s), \se_4(x_d), \se_8(x_a), \se(x_m), \se_8(x_g)) \\
+  \se(x \in \mathbb{O}) &\equiv \se(O(x_\mathbf{o}))\concat\se(x_l, x_k)\concat\se(\var{x_\mathbf{a}})\\
```

Comment:
```suggestion
  \se(x \in \mathbb{O}) &\equiv \se(O(x_\mathbf{o}), x_l, x_k, \var{x_\mathbf{a}})\\
```

Rendered:
![Screenshot_2025-03-02_07-33-45](https://github.com/user-attachments/assets/6614af5f-aa21-4f24-b617-221ee50f5cb0)

Is not required to have separate calls to the serialization operator as all items are using the same